server/server.go

```diff
@@ -68,11 +68,12 @@ func (keys *DefaultKeystore) Get(op *gokeyless.Operation) (crypto.Signer, bool)
 	if ski.Valid() {
 		priv, found := keys.skis[ski]
 		if found {
+			log.Infof("fetch key with SKI: %02x", ski)
 			return priv, found
 		}
 	}
 
-	log.Infof("Couldn't look up key for %s.", op)
+	log.Errorf("couldn't fetch key for %s.", op)
```

Comment:
do you mean that will be a lot of errors? Or a single errorf will kill the server?